### ISSUE TITLE
Depth masks seem to be influencing SSAO

### ISSUE THREAD

**Issue by JonnyOThan** [FirstPersonKSP/FreeIva#446]
![image](https://github.com/user-attachments/assets/b26af6a6-cb41-47ef-b881-0d37b63873e1)

reports so far are TUFX + deferred, MultiScaleVolumetricObscurance (non-default)

**Comment by JonnyOThan** [comment]
waiting on TUFX release: https://github.com/KSPModStewards/TUFX/pull/31
